### PR TITLE
Fixed vg image build issue

### PR DESCRIPTION
**Pull request by cmarkello**

### CODE REVIEW

**Comment by cmarkello** [comment]
@hannes-ucsc @fnothaft Ready for review.

File: vg/build/Dockerfile [hunk 1]
```diff
@@ -1,24 +1,25 @@
...
-	make \
-	protobuf-compiler \
-	libprotoc-dev \
-	libjansson-dev \
-	libbz2-dev \
-	libncurses5-dev \
-	automake libtool jq samtools curl unzip redland-utils \
-	librdf-dev cmake pkg-config wget bc gtk-doc-tools raptor2-utils rasqal-utils bison flex
+RUN apt-get update && \
+	apt-get update &&\
```
Comment (hannes-ucsc) [inline vg/build/Dockerfile]: Just noticing that now: Why run update twice?

Is this change part of the fix? If not, can we drop it or put it into a separate commit?

File: vg/build/Dockerfile [hunk 2]
```diff
@@ -1,24 +1,25 @@
...
+	apt-get install -y \
+		build-essential \
+        gcc-4.9 \
```
Comment (hannes-ucsc) [inline vg/build/Dockerfile]: The indents on this line and the ones below should align with the line above.

File: vg/build/Dockerfile (hunk 1 above)
Comment (hannes-ucsc) [reply vg/build/Dockerfile]: BTW, I prefer having a continued line start with an operator, as in 

```
RUN apt-get update \
    && apt-get upgrade \
    && apt-get install -y \
        build-essential \
        …
```

The reasoning is that having the operator/separator at the beginning of the continuing line creates only two sets of lines. The first line is one set, the subsequent lines form the second set. With the separator at the end of a continued line, there are three distinct sets: 1) the first line, 2) the last line and 3) the lines in between. This makes appending lines in future commits more error prone and creates larger diffs because the old last line needs to be changed.

File: vg/build/Dockerfile [hunk 3]
```diff
@@ -28,4 +29,4 @@ RUN git checkout 8a28f7df06989342c89fa0e9c729db64257188ba
 RUN make get-deps
 
 # Build vg 
-RUN ./source_me.sh && make static
+RUN . ./source_me.sh && make static
```
Comment (hannes-ucsc) [inline vg/build/Dockerfile:31]: This is the fix, I suppose?

**Review by hannes-ucsc — CHANGES_REQUESTED** [review]

**Comment by hannes-ucsc** [comment]
@cmarkello, from now on, @cket will be reviewing PRs against this project.

**Comment by hannes-ucsc** [comment]
In addition to @fnothaft, that is.

File: vg/build/Dockerfile [hunk 4]
```diff
@@ -1,24 +1,25 @@
-FROM ubuntu:14.04
+FROM ubuntu:15.10
```
Comment (cmarkello) [inline vg/build/Dockerfile:1]: It's actually this change which fixes the gcc-4.9 build issue.
I'll fixup the other extraneous stuff.

**Comment by cmarkello** [comment]
@hannes-ucsc @cket @fnothaft Ready for review

**Review by hannes-ucsc — APPROVED** [review]

**Comment by hannes-ucsc** [comment]
@cket, merge at will.

**Review by fnothaft — APPROVED** [review]
